platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/UnsignedEvent.java

```diff
@@ -161,53 +169,158 @@ public BaseEventHashedData(
                 .toArray(new ConsensusTransactionImpl[0]);
     }
 
-    @Override
-    public int getMinimumSupportedVersion() {
-        return ClassVersion.BIRTH_ROUND;
+    /**
+     * Serialize the event for the purpose of creating a hash.
+     * Since events are being migrated to protobuf, calculating the hash of an event will change.
+     *
+     * @param out the stream to which this object is to be written
+     * @throws IOException if unsupported payload types are encountered
+     */
+    public void serializeLegacyHashBytes(final SerializableDataOutputStream out) throws IOException {
+        out.writeLong(CLASS_ID);
+        serialize(out);
     }
 
-    @Override
+    /**
+     * Serialize the event
+     *
+     * @param out the stream to which this object is to be written
+     *
+     * @throws IOException if unsupported payload types are encountered
+     */
     public void serialize(final SerializableDataOutputStream out) throws IOException {
```

Comment:
At first glance, I'm not seeing any unit tests for the serialize / deserialize methods.

Do these exist somewhere I didn't see? If not, I think they should.

Reply:
It was tested with `SerializationTests` added some more (for edge cases) in `PlatformEventTest`